clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -646,14 +659,49 @@ check_duplicate_syscall_with_same_pc()
     return true;
 }
 
+bool
+check_rseq_side_exit_discontinuity()
+{
+    // Negative test: Missing instructions in a basic block due to rseq side exit.
+    instr_t *store = XINST_CREATE_store(GLOBAL_DCONTEXT, OPND_CREATE_MEMPTR(REG2, 0),
+                                        opnd_create_reg(REG1));
+    instr_t *move1 =
+        XINST_CREATE_move(GLOBAL_DCONTEXT, opnd_create_reg(REG1), opnd_create_reg(REG2));
+    instr_t *move2 =
+        XINST_CREATE_move(GLOBAL_DCONTEXT, opnd_create_reg(REG1), opnd_create_reg(REG2));
+    instr_t *jmp =
+        XINST_CREATE_jump_cond(GLOBAL_DCONTEXT, DR_PRED_EQ, opnd_create_instr(move2));
+
+    // Create an instrlist_t*.
```

Comment:
These comments are not really helpful IMHO. It's clear that the code below creates an instrlist, or a std::vector (for the comment below). Maybe improve the comment or just remove it.

Reply:
Ack. Removed it.